src/libs/cluster-utils.js

```diff
@@ -51,13 +51,9 @@ export const runtimeConfigCost = config => {
   ])
 }
 
-export const persistentDiskCostMonthly = config => {
-  return config.size * monthlyStoragePrice
-}
-
-export const persistentDiskCost = config => {
-  return config.size * storagePrice
-}
+export const [persistentDiskCost, persistentDiskCostMonthly] = _.map(price => ({ size, status }) => {
```

Comment:
This makes a lot of sense to me, and I like this change. It seems clear enough to me.